src/lib/comparison.ts

```diff
@@ -19,12 +49,14 @@ export class Comparison<Location> {
    * After creation, first add files to the index which can then be queried.
    *
    * @param tokenizer A tokenizer for the correct programming language
-   * @param hashFilter An optional HashFilter to filter the hashes returned by
-   * the rolling hash function.
+   * @param fragmentOptions The options used to filter based on the fragment count. For a more detailed explanation see
+   * [[ComparisonOptions]]. If this options is not used then no filtering will occur.
    */
-  constructor(tokenizer: Tokenizer<Location>, hashFilter?: HashFilter) {
+  constructor({ tokenizer, hashFilter, noFilter, filterOptions }: ComparisonOptions<Location>) {
```

Comment:
Sorry, my previous comment probably wasn't clear. I didn't want you to bring the tokenizer inside the options hash, but bring the maxFragment and filterFragmentByPercentage to a lower level (and get rid of filter options).